rotkehlchen/chain/ethereum/defi/protocols.py

```diff
@@ -45,6 +45,7 @@ class Protocol(NamedTuple):
     'enzyme': Protocol('Enzyme', 'enzyme.svg'),
     'yearn_vaults': Protocol('yearn.finance &bull; Vaults', 'yearn_vaults.svg'),
     'yearn_vaults_v2': Protocol('yearn.finance &bull; Vaults (v2)', 'yearn_vaults.svg'),
+    'yearn_vaults_v3': Protocol('yearn.finance &bull; Vaults (v3)', 'yearn_vaults.svg'),
```

Comment:
This is not needed. We have this list for the zerionSDK that is a contract that returned balances. It is at https://etherscan.io/address/0x06FE76B2f432fdfEcAEf1a7d4f6C3d41B5861672#readContract . You can see the method `getProtocolNames`. THis list is just a copy of them